<!DOCTYPE HTML>
<!--
	Spectral by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
	<link rel="icon" href="images/Forge-Icon.png">
		<title>Gareth Difford</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<meta name="description" content="Gareth Difford, Phenotype Forgem PhenotypeForge, Phenomics, NMBU" />
	</head>
	<body class="landing is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header" class="alt">
						<h1><a href="Index.html">Gareth Difford</a></h1>
						<nav id="nav">
							<ul>
								<li class="special">
									<a href="#menu" class="menuToggle"><span>Menu</span></a>
									<div id="menu">
										<ul>
											<li><a href="Index.html">Home</a></li>
											<li><a href="PhenotypeForge.html">PhenotypeForge</a></li>
											</ul>
									</div>
								</li>
							</ul>
						</nav>
					</header>

				<!-- Banner -->
					<section id="banner">
						<div class="inner">
							<h2>Gareth Difford</h2>
							<p>Understand the World Through Phenomics</p>
							<ul class="actions special">
								<li><a href="#one" class="button">About Me</a></li>
								<li><a href="#two" class="button">Research</a></li>
								<li><a href="#three" class="button">Teaching</a></li>
								<li><a href="#four" class="button">Publications</a></li>
								<li><a href="PhenotypeForge.html" class="button">Phenotype Forge</a></li>
							</ul>							
							</div>
							<a href="#one" class="more scrolly">Learn More</a>
					</section>
					
							
					
					<!-- One -->
					<section id="one" class="wrapper style1 special">
						<div class="inner">
							<header class="major">
								<h2>Research Focus</h2>
								<p> I record traits on populations of animals to unlock their genetic potential.<br /> 
								If it is too expensive or difficult or invasive to measure, then I am interested <br /> 
								in developing a cheaper, faster or less-invasive alternative. I am a Tenure Track <br />
								Scientist and the Norwegian University of Life Sciences & I lead the Phenotype Forge. <br />
								We generate & handle vast, diverse and continuous datastreams & convert them into phenotypes. <br /> 
								</p>

								<div class="box alt">
								<div class="row gtr-50 gtr-uniform">
											
											<div class="col-4"><span class="image fit"><img src="images/DKC_bin.jpg" alt="" /></span></div>
											<div class="col-4"><span class="image fit"><img src="images/PACs.jpg" alt="" /></span></div>
											<div class="col-4"><span class="image fit"><img src="images/GFD3.jpg" alt="" /></span></div>
											<div class="col-4"><span class="image fit"><img src="images/Hyperspectral_fat3.jpg" alt="" /></span></div>
											<div class="col-4"><span class="image fit"><img src="images/SHF_bin.jpg" alt="" /></span></div>
											<div class="col-4"><span class="image fit"><img src="images/live.jpg" alt="" /></span></div>
											
										</div>
									</div>
							
						</div>
					</section>
			
			
		<!-- Two -->
					<section id="two" class="wrapper alt style2">
					<section class="spotlight">
							<div class="image"><img src="images/Salmon_non.jpg" alt="" /></div><div class="content">
								<h2>DigiFishent Project</h2>
								<p> Feed remains the highest production cost in Atlantic <br />
								salmon,	it is also responsible for more than 80% of the<br />
								carbon footprint. Despite shifts from largely fish based<br />
								feed ingredients to plant based ingredients. This has lead <br />
								to plummeting levels of EPA and DHA in Atlantic salmon, <br />
								the healthy Omega-3 fatty acids that make it desirable. <br />						
								This provides an opportunity of reduce the carbon footprint<br />
								and increase profitability, through increasing feed efficiency. <br />
								Measuring feed efficiency is extremely challenging as individual <br />
								feed intake records are almost impossible to record. Using new<br />
								digital technologies we will record feed intake in Atlantic <br />
								salmon. This will be coupled with Nofima's Raman spectroscopy<br />
								for recording Omega-3 fatty acids in the fish. This is a <br />
								Norwegian Research Council prestigious FriPro Young Talent
								project. In collaboration with Nofima & AquaGen <br />
								</p>
							</div>
							</section>
							<section class="spotlight">
							<div class="image"><img src="images/Raman.jpg" alt="" /></div><div class="content">
								<h2>Vibrational Spectra as Phenotypes</h2>
								<p> In many applications we want to know more about the<br>
								composition of our samples inorder to breeding for higher<br>
								product quality or efficiency. Vibrational spectroscopy offers<br />
								a way to investigate chemical composition of complex biological<br />
								mixtures like meat and milk. When coupled with rapid delivery<br />
								methods such as robots or conveyor belts, this becomes a <br />
								highthrouhg-put phenotyping method of choice. We have found <br />
								that Raman shifts & Near-Infrared spectra recorded on Atlantic<br />
								salmon fillets are significantly heritable & very good predicters<br />
								of total fat. Currently, we are pushing this frontier & <br />
								investigating detailed fatty acid composition. <br /> 
								In collaboration with materials and processing group of Nofima.<br /> 
								<a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-021-00605-6">Read More about heritable spectra</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8997921/" >& Fatty acids</a> 
								</p>
							</div>
						</section>
					<section id="two" class="wrapper alt style2">
					<section class="spotlight">
							<div class="image"><img src="images/Rumen_tube_2.jpg" alt="" /></div><div class="content">
								<h2>Microbiability & Heritability</h2>
								<p> Breakthroughs in sequencing technologies has opened up<br />
								the world of microbiomics. Ruminants have the remarkable <br />
								ability to digest (human inedible) fibre is grass and <br />
								crop residues. This ability is encoded for in the genomes <br />
								of their rumen microbiome. We have found that the relative <br />
								abundance of certain bacteria and archea is governed by a <br />
								significant proportion of the host cows additive genetics.<br />
								Remarkably,	the variation between cows for traits like <br />
								methane emissions, feed efficiency, ketosis and <br />
								milk fatty acids can all be explained  partly by the rumen<br />
								microbial composition. We coined the term "Microbiability"<br />
								in analogy to heritability to explain this phenomena. <br />
								In collaboration with Aarhus University & Technical University of Denmark <br />
								<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007580">Microbiability & Methane</a>
								<a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0464-8">Microbiability & milk fatty acids</a>
								</p>
							</div>
							</section>
					
				<!-- Three -->
					<section id="three" class="wrapper style3 special">
						<div class="inner">
							<header class="major">			
								<h2>Teaching</h2>
								</header>
								<h3>Post-graduate</h3>
								<h4>BIN302</h4>
								<p>I am course responsible for the post graduate course BIN302 Advanced analysis for high throughput phenotyping<br>
								and precision farming. In BIN302 we teach students at NMBU how to handle and process the wide and diverse datasets<br>
								coming from sensors in 	time series, vibrational spectra, images & video. We also teach how to use machine & deep<br>
								learning tools for predicting phenotypes. Lastly & most importantly we teach students how to benchmark need phenotypes <br>
								against current or existing gold standards to make informed decisions on their use.<br>
								<h4>NOVA410</h4>
								In this PhD course focussed on Biological consequences of selection, I teach on theoretical and practical use of <br>
								selection indices for predicting multitrait response to selection. We apply this to the emission and feed efficiency <br>
								traits like methane production and residual feed intake in dairy cattle and other important species. <br>
								<br>
								<h4>Guest Lecturing</h4>
								<ul>
								<li>2023. Training School in Experimental Design & Statistical Analysis of Biomedical and Bioscience Experiments. UiB</li>
								<li>2023. HFA300 Livestock breeding and conservation plans. NMBU</li>
								<li>2023. HFX315 Smart Livestock Production. NMBU</li>
											</ul>
								<h3>Under-graduate</h3>
								<h4>Guest Lecturing</h4>
								<ul>
								<li>2023. HFA200 General Livestock Breeding. NMBU </li>
								<li>2023. AQB250 Sustainable aquaculture - breeding and genetics</li>
								</ul>
								</p>
								<hr />
						</div>
					</section>
					
					
					<!-- Four -->
					<section id="four" class="wrapper alt style2">
		<h2>Publications</h2>
		<h2>total #n=134, h-index 37, i10-index 19 (Google Scholar <a href="https://scholar.google.co.uk/citations?user=8FnwHnMAAAAJ&hl=en"> Profile</a>)</h2>
		
<h3>Publications - The Omics</h3>										
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Title</th>
													<th>Link</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td><u>Difford, G. F.</u>, Plichta, D. R., Løvendahl, P., Lassen, J., Noel, S. J., Højberg, O., ... & Guldbrandtsen, B. (2018). Host genetics and the rumen microbiome jointly associate with methane emissions in dairy cows. PLoS genetics, 14(10), e100750. </td>
													<td><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007580"> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.</u>, Haugen, J.E., Aslam, M.L., Johansen, L.H., Breiland, M.W., Hillestad, B., Baranski, M., Boison, S., Moghadam, H. and Jacq, C., 2022. Variation in volatile organic compounds in Atlantic salmon mucus is associated with resistance to salmon lice infection. Scientific Reports, 12(1), p.4839. </td>
													<td><a href="https://www.nature.com/articles/s41598-022-08872-z"> Link</a></td>
												</tr>
												<tr>
													<td>Zhu, Z., <u>Difford, G.F.</u>, Noel, S.J., Lassen, J., Løvendahl, P. and Højberg, O., 2021. Stability assessment of the rumen bacterial and archaeal communities in dairy cows within a single lactation and its association with host phenotype. Frontiers in Microbiology, 12, p.636223.</td>
													<td><a href="https://www.frontiersin.org/articles/10.3389/fmicb.2021.636223/full"> Link</a></td>
												</tr>
												<tr>
													<td>Khaw, H.L., Gjerde, B., Boison, S.A., Hjelle, E. & <u>Difford, G.F.</u>, 2021. Quantitative Genetics of Smoltification Status at the Time of Seawater Transfer in Atlantic Salmon (Salmo Salar). Frontiers in genetics, </td>
													<td><a href="https://www.frontiersin.org/articles/10.3389/fgene.2021.696893/full"> Link</a></td>
												</tr>
												<tr>
													<td>Zhang, , Q., <u>Difford, G.</u>, Sahana, G., Løvendahl, P., Lassen, J., Lund, M.S., Guldbrandtsen, B. and Janss, L., 2020. Bayesian modeling reveals host genetics associated with rumen microbiota jointly influence methane emission in dairy cows. The ISME journal, 14(8), pp.2019-2033. </td>
													<td><a href="https://www.nature.com/articles/s41396-020-0663-x"> Link</a></td>
												</tr>
												<tr>
													<td>Gebreyesus, G., <u>Difford,</u> G.F., Buitenhuis, B., Lassen, J., Noel, S.J., Højberg, O., Plichta, D.R., Zhu, Z., Poulsen, N.A., Sundekilde, U.K. and Løvendahl, P., 2020. Predictive ability of host genetics and rumen microbiome for subclinical ketosis. Journal of dairy science, 103(5), pp.4557-4569.  </td>
													<td><a href="https://www.sciencedirect.com/science/article/pii/S0022030220302150"> Link</a></td>
												</tr>
												<tr>
													<td>Buitenhuis, B., Lassen, J., Noel, S.J., Plichta, D.R., Sørensen, P., <u>Difford, G.F.</u> & Poulsen, N.A., 2019. Impact of the rumen microbiome on milk fatty acid composition of Holstein cattle. Genetics Selection Evolution, 51(1), pp.1-8 </td>
													<td><a href="https://gsejournal.biomedcentral.com/articles/10.1186/s12711-019-0464-8"> Link</a></td>
												</tr>
												<tr>
													<td>Zhu, Z., Kristensen, L., <u>Difford, G.F.</u>, Poulsen, M., Noel, S.J., Al-Soud, W.A., Sørensen, S.J., Lassen, J., Løvendahl, P. and Højberg, O., 2018. Changes in rumen bacterial and archaeal communities over the transition period in primiparous Holstein dairy cows. Journal of dairy science, 101(11), pp.9847-9862. </td>
													<td><a href="https://www.journalofdairyscience.org/article/S0022-0302(18)30799-9/fulltext"> Link</a></td>
												</tr>
												<tr>
													<td> <u>Difford, G.F.</u>, Løvendahl, P., Lassen, J., Guldbrandtsen, B. and Sahana, G., 2018. Microbiability–new insights into (genetic) modelling methane emissions of cattle. In Proc. of the World Congress on Genetics Applied to Livestock Production (Vol. 11, pp. 405-10). </td>
													<td><a href="https://pure.au.dk/portal/files/128030082/microbiability_new_insights_genetic_modelling_methane_emissions_cattle.pdf"> Link</a></td>
												</tr>
												<tr>
													<td>Zhu, Z., Noel, S.J., <u>Difford, G.F.</u>, Al-Soud, W.A., Brejnrod, A., Sørensen, S.J., Lassen, J., Løvendahl, P. and Højberg, O., 2017. Community structure of the metabolically active rumen bacterial and archaeal communities of dairy cows over the transition period. PLoS One, 12(11), p.e0187858.. </td>
													<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0187858"> Link</a></td>
												</tr>
											</tbody>
											
											</table>
									</div><tr>
	<h3>Publications- Sensors</h3>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Title</th>
													<th>Link</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td><u>Difford, G. F.</u>, Løvendahl, P., Veerkamp, R. F., Bovenhuis, H., Visker, M. H. P. W., Lassen, J., & De Haas, Y. (2020). Can greenhouse gases in breath be used to genetically improve feed efficiency of dairy cows? Journal of dairy science, 103(3), 2442-2459.  </td>
													<td><a href="https://www.journalofdairyscience.org/article/S0022-0302(20)30017-5/fulltext"> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G. F.</u>, Lassen, J., & Løvendahl, P. (2016). Interchangeability between methane measurements in dairy cows assessed by comparing precision and agreement of two non-invasive infrared methods. Computers and Electronics in Agriculture, 124, 220-226.  </td>
													<td><a href="https://www.sciencedirect.com/science/article/pii/S0168169916301260"> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G. F.</u>, Olijhoek, D. W., Hellwing, A. L. F., Lund, P., Bjerring, M. A., De Haas, Y., ... & Løvendahl, P. (2018). Ranking cows’ methane emissions under commercial conditions with sniffers versus respiration chambers. Acta Agriculturae Scandinavica, Section A—Animal Science, 68(1), 25-32.  </td>
													<td><a href="https://www.tandfonline.com/doi/abs/10.1080/09064702.2019.1572784"> Link</a></td>
												</tr>
												<tr>
													<td>Manzanilla-Pech, C.I.V., Gordo, D.M., <u>Difford, G.F.</u>, Pryce, J.E., Schenkel, F., Wegmann, S., Miglior, F., Chud, T.C., Moate, P.J., Williams, S.R.O. and Richardson, C.M., 2021. Breeding for reduced methane emission and feed-efficient Holstein cows: An international response. Journal of Dairy Science, 104(8), pp.8983-9001.  </td>
													<td><a href="https://www.journalofdairyscience.org/article/S0022-0302(21)00597-X/fulltext"> Link</a></td>
												</tr>
												<tr>
													<td>Wethal, K. B., <u>G. F. Difford</u>, K. Winnberg, E. Norberg, and B. Heringstad. "Heritability of methane emission in Norwegian Red cows based on measures from GreenFeed in commercial herds." In Proceedings of 12th (WCGALP) pp. 176-178. Wageningen Academic Publishers, 2022. </td>
													<td><a href="https://www.wageningenacademic.com/doi/abs/10.3920/978-90-8686-940-4_32"> Link</a></td>
												</tr>
												<tr>
													<td>Manzanilla-Pech, C.I.V., <u>Difford, G.F.</u>, Løvendahl, P., Stephansen, R.B. and Lassen, J., 2022. Genetic (co-) variation of methane emissions, efficiency, and production traits in Danish Holstein cattle along and across lactations. Journal of Dairy Science, 105(12), pp.9799-9809.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Manzanilla-Pech, C.I.V., <u>Difford, G.F.</u>, Sahana, G., Romé, H., Løvendahl, P. and Lassen, J., 2022. Genome-wide association study for methane emission traits in Danish Holstein cattle. Journal of Dairy Science, 105(2), pp.1357-1368.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Manzanilla-Pech, C.I.V., Gordo, D., <u>Difford, G.F.</u>, Løvendahl, P. and Lassen, J., 2020. Multitrait genomic prediction of methane emissions in Danish Holstein cattle. Journal of dairy science, 103(10), pp.9195-9206.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Olijhoek, D.W., <u>Difford, G.F.</u>, Lund, P. and Løvendahl, P., 2020. Phenotypic modeling of residual feed intake using physical activity and methane production as energy sinks. Journal of dairy science, 103(8), pp.6967-6981.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.</u>, Boison, S.A., Khaw, H.L. and Gjerde, B., 2020. Validating non-invasive growth measurements on individual Atlantic salmon in sea cages using diode frames. Computers and Electronics in Agriculture, 173, p.105411. </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Garnsworthy, P.C., <u>Difford, G.F.</u>, Bell, M.J., Bayat, A.R., Huhtanen, P., Kuhla, B., Lassen, J., Peiren, N., Pszczola, M., Sorg, D. and Visker, M.H., 2019. Comparison of methods to measure methane for use in genetic evaluation of dairy cattle. Animals, 9(10), p.837. </td>
													<td><a href=""> Link</a></td>
												</tr>
																							
													<td>Szalanski, M., Kristensen, T., <u>Difford, G.</u>, Lassen, J., Buitenhuis, A.J., Pszczola, M. and Løvendahl, P., 2019. Enteric methane emission from Jersey cows during the spring transition from indoor feeding to grazing. Journal of dairy science, 102(7), pp.6319-6329.</td>
													<td><a href=""> Link</a></td>
												</tr>
																								
													<td>Zetouni, L., <u>Difford, G.F.</u>, Lassen, J., Byskov, M.V., Norberg, E. and Løvendahl, P., 2018. Is rumination time an indicator of methane production in dairy cows?. Journal of dairy science, 101(12), pp.11074-11085.  </td>
													<td><a href=""> Link</a></td>
												</tr>
																								
													<td>Sorg, D., <u>Difford, G.F.</u>, Mühlbach, S., Kuhla, B., Swalve, H.H., Lassen, J., Strabel, T. and Pszczola, M., 2018. Comparison of a laser methane detector with the GreenFeed and two breath analysers for on-farm measurements of methane emissions from dairy cows. Computers and electronics in agriculture, 153, pp.285-294. </td>
													<td><a href=""> Link</a></td>
												</tr>
											</tbody>
											
											</table>
									</div><tr>												
								</section>
								
								
								<h3>Publications - Vibrational Spectra</h3>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Title</th>
													<th>Link</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td> <u>Difford, G.F.</u>, Horn, S.S., Dankel, K.R., Ruyter, B., Dagnachew, B.S., Hillestad, B., Sonesson, A.K. and Afseth, N.K., 2021. The heritable landscape of near-infrared and Raman spectroscopic measurements to improve lipid content in Atlantic salmon fillets. Genetics Selection Evolution, 53(1), pp.1-11.  </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Shetty, N., <u>Difford, G.</u>, Lassen, J., Løvendahl, P., & Buitenhuis, A. J. (2017). Predicting methane emissions of lactating Danish Holstein cows using Fourier transform mid-infrared spectroscopy of milk. Journal of dairy science, 100(11), 9052-9060.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.</u>, Horn, S.S., Dankel, K.R., Ruyter, B., Dagnachew, B.S., Hillestad, B., Sonesson, A.K. and Afseth, N.K., 2021. The heritable landscape of near-infrared and Raman spectroscopic measurements to improve lipid content in Atlantic salmon fillets. Genetics Selection Evolution, 53, pp.1-11.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Afseth, N.K., Dankel, K., Andersen, P.V., <u>Difford, G.F.</u>, Horn, S.S., Sonesson, A., Hillestad, B., Wold, J.P. and Tengstrand, E., 2022. Raman and near Infrared Spectroscopy for Quantification of Fatty Acids in Muscle Tissue—A Salmon Case Study. Foods, 11(7), p.962.</td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Horn, S.S., Aslam, M.L., <u>Difford, G.F.</u>, Tsakoniti, K., Karapanagiotis, S., Gulzari, B., Bastiaansen, J.W.M., Penaloza, C., Houston, R., Ruyter, B. and Sonesson, A.K., 2022. Genetic parameters of fillet fatty acids and fat deposition in gilthead seabream (Sparus aurata) using the novel 30 k Medfish SNP array. Aquaculture, 556, p.738292. </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.,</u> Díaz-Gil, C., Sánchez-Moya, A., Aslam, M.L., Horn, S.S., Ruyter, B., Herlin, M., Lopez, M. and Sonesson, A.K., 2021. Genomic and Phenotypic Agreement Defines the Use of Microwave Dielectric Spectroscopy for Recording Muscle Lipid Content in European Seabass (Dicentrarchus labrax). Frontiers in Genetics, 12, p.671491.</td>
													<td><a href=""> Link</a></td>
												</tr>
												</tbody>
											
											</table>
									</div><tr>												
								</section>
								
								<h3>Publications - Image Analysis</h3>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Title</th>
													<th>Link</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Lindberg, S.K., Durland, E., Heia, K., Noble, C., Alvestad, R. and <u>Difford, G.F.</u>, 2023. Digital scoring of welfare traits in Atlantic salmon (Salmo salar L.)-a proof of concept study quantifying dorsal fin haemorrhaging via. Frontiers in Animal Science, 4, p.33.  </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.,</u> Hatlen, B., Heia, K., Bæverfjord, G., Eckel, B., Gannestad, K.H., Romarheim, O.H., Lindberg, S.K., Norris, A.T., Sonesson, A.K. and Gjerde, B., 2023. Digital phenotyping of individual feed intake in Atlantic salmon (Salmo salar) with the X-ray method and image analysis. Frontiers in Animal Science, 4, p.28. </td>
													<td><a href=""> Link</a></td>
												</tr>
												</tbody>
											
											</table>
									</div><tr>												
								</section>
								
								<h3>Publications - Reviews & Overviews</h3>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Title</th>
													<th>Link</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Løvendahl, P., <u>Difford, G. F.</u>, Li, B., Chagunda, M. G. G., Huhtanen, P., Lidauer, M. H., ... & Lund, P. (2018). Selecting for improved feed efficiency and reduced methane emissions in dairy cattle. animal, 12(s2), s336-s349.</td>
													<td><a href=""> Link</a></td>
												</tr>
													<td>Lassen, J. and <u>Difford, G.F.</u>, 2020. Genetic and genomic selection as a methane mitigation strategy in dairy cattle. Animal, 14, pp.s473-s483.  </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Robinson, Nicholas A., Diego Robledo, Lene Sveen, Rose Ruiz Daniels, Aleksei Krasnov, Andrew Coates, Ye Hwa Jin et al. "Applying genetic technologies to combat infectious diseases in aquaculture." Reviews in Aquaculture 15, no. 2 (2023): 491-535. </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td><u>Difford, G.F.,</u> Lindberg, S.K., Heia, K., Hatlen, B., Durland, E., Gjerde, B., Alvestad, R., Mota, V., Noble, C., Norris, A. and Boison, S., 2022, December. Digital phenotyping to improve Atlantic salmon feed efficiency and welfare. In Proceedings of 12th World Congress on Genetics Applied to Livestock Production (WCGALP) Technical and species orientated innovations in animal breeding, and contribution of genetics to solving societal challenges (pp. 569-572). Wageningen Academic Publishers.  </td>
													<td><a href=""> Link</a></td>
												</tr>
												<tr>
													<td>Sonesson, A.K., Hatlen, B., Gjerde, B., <u>Difford, G.F.</u>, Brunsvik, P., Meuwissen, T.H.E. and Norris, A., 2022, December. Genetic analysis of feed efficiency in Atlantic salmon. In Proceedings of 12th World Congress on Genetics Applied to Livestock Production (WCGALP) Technical and species orientated innovations in animal breeding, and contribution of genetics to solving societal challenges (pp. 284-287). Wageningen Academic Publishers. </td>
													<td><a href=""> Link</a></td>
												</tr>
												</tbody>
											
											</table>
									</div><tr>												
								</section>
								
								
								
											
				<!-- CTA -->
					<section id="cta" class="wrapper style4">
						<div class="inner">
							<header>
					<blockquote> "In the age of the genotype, phenotype is king" Professor Mike Coffey </blockquote>
												</header>
							<ul class="actions stacked">
					
							</ul>
						</div>
					</section>

				<!-- Footer -->
					<footer id="footer">
						<ul class="icons">
							<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="#" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul>
						<ul class="copyright">
							<li>&copy; Untitled</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
						</ul>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>